### PR TITLE
Translate some more strings

### PR DESCRIPTION
**Pull request by andrewshadura**

### CODE REVIEW

File: src/translations/en.js [hunk 1]
```diff
@@ -78,24 +78,24 @@ export default {
         fork: 'zvoľte cestu',
         'end of road': 'pokračujte',
         // 'use lane':
-        continue: 'pokračujte',
-        roundabout: 'vojdite na kruhový objazd',
-        rotary: 'vojdite na okružnú cestu',
-        'roundabout turn': 'na kruhovom objazde odbočte',
+        continue: 'contrinue',
```
Comment (zdila) [inline src/translations/en.js]: contrinue? ;-)

Comment (andrewshadura) [reply src/translations/en.js]: oargh.

Comment (andrewshadura) [reply src/translations/en.js]: resubmitted

**Comment by zdila** [comment]
Vdaka!

**Pull request merged**